### DISCUSSION TITLE
How to uncheck virtual select dynamically?

### DISCUSSION THREAD

**Discussion by vasudedakiya** [sa-si-dev/virtual-select#241] [Q&A] [answered]
I have a tags in my code. that if virtual select option check at that time that tag with close button option generate.
and if i click on that close button than i want virtual select that item also uncheck automatically.

How can I do that?

**Comment by sa-si-dev** [comment, marked as answer]
You should call [setValue](https://sa-si-dev.github.io/virtual-select/#/methods?id=setvalue) method with latest values without removed value.

**Reply by vasudedakiya** [reply]
Can I uncheck option by using it?

**Reply by sa-si-dev** [reply]
> Can I uncheck option by using it?

Yes. Let's say we have current value as `[1,2,3]`
To remove the option `2`, call `setValue` method with new value `[1,3]`

**Reply by vasudedakiya** [reply]
if I want to uncheck all then if I'm passing empty array then it will uncheck all right?

**Reply by sa-si-dev** [reply]
> if I want to uncheck all then if I'm passing empty array then it will uncheck all right?

Yes
